### DISCUSSION TITLE
What is the proper way of including color resources from another file in App.axaml?

### DISCUSSION THREAD

**Discussion by JaoHundred** [AvaloniaUI/Avalonia#13636] [Q&A] [answered]
Hi, I'm trying to include a resource file into App.axaml. I've already read through https://docs.avaloniaui.net/docs/next/guides/styles-and-resources/resources and I still dont understand how should I be doing this. Could someone provide a better example to this?

App.axaml file:
![image](https://github.com/AvaloniaUI/Avalonia/assets/43298295/29f7efce-597e-4592-a8a6-baf1ca073185)

Colors.axaml file:
![image](https://github.com/AvaloniaUI/Avalonia/assets/43298295/28c186b6-99a2-4234-8b8d-563e48044ab0)

**Comment by workgroupengineering** [comment, marked as answer]
See

https://github.com/AvaloniaUI/Avalonia.Samples/blob/94eae1a02083a5a146f1961947b66d81c0f9b8cd/src/Avalonia.Samples/CustomControls/RatingControlSample/App.axaml#L6-L13

**Reply by JaoHundred** [reply]
I've tried everything and forgot to try exactly that, thank you.

**Comment by maxkatz6** [comment]
See ResourceInclude, we should have documentation on it

**Reply by maxkatz6** [reply]
https://docs.avaloniaui.net/docs/next/guides/styles-and-resources/resources
